util.sh

```diff
@@ -126,57 +127,46 @@ function my_travis_wait() {
     timeout=20
   fi
 
-  # Show command in console before running
-  echo -e "\e[34m$ $@\e[0m"
-
-  my_travis_wait_impl $timeout "$@"
-}
-
-#######################################
-function my_travis_wait_impl() {
-  local timeout=$1
-  shift
-
-  local cmd="$@"
-  local log_file=my_travis_wait_$$.log
+  local cmd=$@
+  let "TRAVIS_FOLD_COUNTER += 1"
+  travis_time_start moveit_ci.$TRAVIS_FOLD_COUNTER $cmd
 
-  $cmd 2>&1 >$log_file &
+  # Disable bash's job control messages
+  set +m
+  # Run actual command in background
+  $cmd &
   local cmd_pid=$!
 
-  my_travis_jigger $! $timeout $cmd &
+  travis_jigger $cmd_pid $timeout $cmd &
   local jigger_pid=$!
   local result
 
   {
     wait $cmd_pid 2>/dev/null
     result=$?
+    # if process finished before jigger, stop the jigger too
     ps -p$jigger_pid 2>&1>/dev/null && kill $jigger_pid
-  } || return 1
+  }
 
-  echo -e "\nThe command \"$cmd\" exited with $result."
-  #echo -e "\n\033[32;1mLog:\033[0m\n"
-  cat $log_file
+  echo
+  travis_time_end
 
   return $result
 }
 
 #######################################
-function my_travis_jigger() {
-  # helper method for travis_wait()
+function travis_jigger() {
```

Comment:
i'm nervous there might be a naming conflict with travis' default scripts, but i can't remember if they have a travis_jigger. i found various similar functions though:
https://github.com/matomo-org/travis-scripts/blob/master/travis-helper.sh#L34

something to look out for

Reply:
It worked fine.